### PR TITLE
BaSP-MR-244: Fixed styles and concistency in profile

### PR DESCRIPTION
**Pull request by Gonzarey96**
-Fixed the styles of the 3 profiles (member, trainer and admin) to be consistent with each other.
-Added identification of the user who is visiting his profile.
-Added change to view-only DNI input.

### CODE REVIEW

**Comment by Gonzarey96** [comment]
[[Admin, Member, Trainer] - Profile](https://trello.com/c/IUxqtbdI/244-admin-member-trainer-profile)

**Review by fernandofernandezuy — APPROVED** [review]
Good job Gonza!

File: src/Components/Admin/Profile/index.jsx [hunk 1]
```diff
@@ -105,78 +105,70 @@ const AdminProfile = () => {
         }}
       />
       <div className={styles.formContainer}>
-        <h2 className={styles.formTitleTwo}>admin data</h2>
+        <h2 className={styles.formTitleTwo}>
+          {editMode
+            ? `${admin?.firstName} ${admin?.lastName}'s Profile`
+            : `${admin?.firstName} ${admin?.lastName}'s Profile`}
```
Comment (lorenzomauro59) [inline src/Components/Admin/Profile/index.jsx]: You should remove the ' 's ' 
```suggestion
            : `${admin?.firstName} ${admin?.lastName} Profile`}
```

File: src/Components/Member/Profile/index.jsx [hunk 2]
```diff
@@ -111,14 +111,18 @@ const MemberForm = () => {
...
       <Toaster
         containerStyle={{
           margin: '10vh 0 0 0'
         }}
       />
       <div className={styles.formContainer}>
-        <h2 className={styles.formTitleTwo}>My Profile</h2>
+        <h2 className={styles.formTitleTwo}>
+          {editMode
+            ? `${user?.firstName} ${user?.lastName}'s Profile`
```
Comment (lorenzomauro59) [inline src/Components/Member/Profile/index.jsx]: same there!

**Review by lorenzomauro59 — APPROVED** [review]
Great job gonzaa! only details in comments

**Pull request merged**